packages/devtools-network-console/src/ui/Authorization/BearerToken.tsx

```diff
@@ -16,38 +20,37 @@ export default function BearerToken(props: IBearerTokenProps) {
     const dispatch = useDispatch();
 
     return (
-        <Stack tokens={{ childrenGap: 'm', padding: 'm' }}>
-            <MessageBar
-                messageBarType={MessageBarType.severeWarning}
-                isMultiline={true}
-                messageBarIconProps={{ iconName: 'Warning' }}
-                styles={{ root: { userSelect: 'none' }}}>
-                These parameters may contain sensitive information. Consider specifying these with environment variables here, and specifying
-                that your environment variables aren't checked in to source control.
-            </MessageBar>
-            <MessageBar
-                messageBarType={MessageBarType.info}
-                messageBarIconProps={{iconName: 'Info'}}
-                styles={{root: { userSelect: 'none'}}}>
-                Environment variable substitutions are not shown here for privacy purposes.
-            </MessageBar>
-            <TextField
-                onChange={e => {
-                    const value = (e.target as HTMLInputElement).value;
-                    dispatch(makeSetBearerTokenAction(props.requestId, value));
-                }}
-                label="Token"
-                value={props.token}
-                styles={{
-                    field: {
-                        fontFamily: 'Consolas, monospace !important',
-                        fontSize: 'small'
-                    },
-                }}
-                underlined
-                multiline
-                autoAdjustHeight
-                />
+        <Stack>
+            <LocalAlert
+                type="severeWarning"
+                textKey="Authorization.Shared.dataSensitiveWarning" />
+            <LocalAlert
+                type="info"
+                textKey="Authorization.Shared.noEnvironmentCalcuations" />
+
+            <Stack horizontal>
+                <div style={{ flexGrow: 0, flexShrink: 0, width: '75px', padding: '15px 0 0 10px', fontWeight: 'bold' }}>
```

Comment:
I have considered that, but such a component would introduce additional complexity because it would need to pass a lot of stuff around (particularly around IDs and whatnot) without introducing a lot of savings.